app/models/source.rb

```diff
@@ -15,6 +15,7 @@ class Source < ApplicationRecord
            :to => :default_endpoint, :allow_nil => true
   validates :name, :presence => true, :allow_blank => false,
             :uniqueness => { :scope => :tenant_id }
+  attribute :source_ref, :string
```

Comment:
This isn't necessary unless you're trying to change the way the attribute is serialized to SQL, rails defines these attributes automatically from the db columns